{"version":"3.7.1","messages":["\nAn error occurred while loading ./spec/no_ruby_errors_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0mprints \u001b[32mdescribe\u001b[0m \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31mNot having any errors and being all green\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m  \u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mcontext\u001b[0m \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mNameError\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0m\u001b[32mit\u001b[0m \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mraises a NameError when encountering undefined barewords\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m        \u001b[0m\u001b[32mexpect\u001b[0m{\u001b[0m\n\u001b[31m          \u001b[0mload \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31m./lib/a_name_error.rb\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m\u001b[0m\n\u001b[31m        \u001b[0m}.to_not raise_error\u001b[0m\n\u001b[31m      \u001b[0m\u001b[32mend\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mend\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mNoMethodError:\u001b[0m\n\u001b[31m  undefined method `prints' for main:Object\u001b[0m\n\u001b[31m  Did you mean?  print\u001b[0m\n\u001b[31m                 printf\u001b[0m\n\u001b[31m                 sprintf\u001b[0m\n\u001b[36m# ./spec/no_ruby_errors_spec.rb:1:in `<top (required)>'\u001b[0m\n","No examples found."],"examples":[],"summary":{"duration":0.00058852,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":1},"summary_line":"0 examples, 0 failures, 1 error occurred outside of examples"}